src/cloudant/account.py

```diff
@@ -148,12 +132,13 @@ def session_logout(self):
 
     def basic_auth_str(self):
         """
-        Compose a basic http auth string, suitable for use with the
+        Composes a basic http auth string, suitable for use with the
         _replicator database, and other places that need it.
 
         TODO: I'm not a huge fan of doing basic auth -- need to
```

Comment:
Moved TODO to a comment inside of the code block so that it will not appear in the documentation.  Done in b19046696b22d6b3fb89e04d7d4b7a86ca7d9c41.